docs/platforms/python/integrations/index.mdx

```diff
@@ -108,6 +110,91 @@ The Sentry SDK uses integrations to hook into the functionality of popular libra
 | <LinkWithPlatformIcon platform="python.tryton"        label="Tryton"          url="/platforms/python/integrations/tryton" />        |                  |
 | <LinkWithPlatformIcon platform="python.wsgi"          label="WSGI"            url="/platforms/python/integrations/wsgi" />          |                  |
 
-## Default Integrations
-
-- [List of Default Integrations](default-integrations/)
+### Default Integrations
+
+- [List of default integrations](default-integrations/)
```

Comment:
I find it somewhat strange that we have a table for all the other integration categories, except this one. Perhaps we can add a table, and then provide the link to the `default_integrations` page for anyone desiring more information about a specific integration (or even better perhaps, we could have each row in the table link to the relevant subsection in the Default Integrations page)?